docs/docs/integrations/llms/llamacpp.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
     "# Llama.cpp\n",
     "\n",
-    "[llama-cpp-python](https://github.com/abetlen/llama-cpp-python) is a Python binding for [llama.cpp](https://github.com/ggerganov/llama.cpp). \n",
+    "[llama-cpp-python](https://github.com/abetlen/llama-cpp-python) is a Python binding for [llama.cpp](https://github.com/ggerganov/llama.cpp).\n",
     "\n",
-    "It supports inference for [many LLMs](https://github.com/ggerganov/llama.cpp), which can be accessed on [HuggingFace](https://huggingface.co/TheBloke).\n",
+    "It supports inference for Meta's [LLaMA](https://github.com/facebookresearch/llama/) models, which can be accessed on [HuggingFace](https://huggingface.co/TheBloke).\n",
```

Comment:
llama.cpp supports many LLMs (I've copied a text from the site). 
"Supported models:

 LLaMA 🦙
 LLaMA 2 🦙🦙
 Falcon
 [Alpaca](https://github.com/ggerganov/llama.cpp#instruction-mode-with-alpaca)
 [GPT4All](https://github.com/ggerganov/llama.cpp#using-gpt4all)
 [Chinese LLaMA / Alpaca](https://github.com/ymcui/Chinese-LLaMA-Alpaca) and [Chinese LLaMA-2 / Alpaca-2](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2)
 [Vigogne (French)](https://github.com/bofenghuang/vigogne)
 https://github.com/ggerganov/llama.cpp/discussions/643#discussioncomment-5533894
 [Koala](https://bair.berkeley.edu/blog/2023/04/03/koala/)
 [OpenBuddy 🐶 (Multilingual)](https://github.com/OpenBuddy/OpenBuddy)
 [Pygmalion/Metharme](https://github.com/ggerganov/llama.cpp#using-pygmalion-7b--metharme-7b)
 [WizardLM](https://github.com/nlpxucan/WizardLM)
 [Baichuan 1 & 2](https://huggingface.co/models?search=baichuan-inc/Baichuan) + [derivations](https://huggingface.co/hiyouga/baichuan-7b-sft)
 [Aquila 1 & 2](https://huggingface.co/models?search=BAAI/Aquila)
 https://github.com/ggerganov/llama.cpp/pull/3187
 [Mistral AI v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1)
 [Refact](https://huggingface.co/smallcloudai/Refact-1_6B-fim)
 https://github.com/ggerganov/llama.cpp/pull/3410
 https://github.com/ggerganov/llama.cpp/pull/3417
 https://github.com/ggerganov/llama.cpp/pull/3553"

Reply:
I agree and reverted the line. The only change is that the link now redirects to the Description of llama.cpp's README